build/configs/rtl8730e/loadable_apps/defconfig

```diff
@@ -1472,7 +1490,9 @@ CONFIG_EXAMPLES_HELLOXX=y
 # Wifi Manager
 #
 # CONFIG_EXAMPLES_TAHI is not set
-# CONFIG_EXAMPLES_WIFIMANAGER_TEST is not set
+CONFIG_EXAMPLES_WIFIMANAGER_TEST=y
```

Comment:
again please avoid default enable. it can be done via menuconfig

Reply:
Done